### ISSUE TITLE
cdvd_info2 fails on some discs

### ISSUE THREAD

**Issue by rs1n** [ps2homebrew/hdl-dump#32]
cdvd_info2 seems to fail to detect:

SCUS97558 (Jak and Daxter - Lost Frontier)
SLUS20079 (Dynasty Warriors 2)

**Comment by bignaux** [comment]
Are you using latest developement release and try with the CLI instead of the GUI ?

**Comment by rs1n** [comment]
I use the CLI. Are there pre-built Win binaries available of the most recent release (don't have any environments to build at the moment)? This is the output when run without arguments using a binary I downloaded some time ago:

hdl_dump-0.9.2 by The W1zard 0f 0z (AKA b...), revisited by AKuHAK
https://bitbucket.org/AKuHAK/hdl-dump w1zard0f07@yahoo.com

Usage:
hdl_dump.exe command arguments

Where command is one of:
query, dump, compare_iin, toc, hdl_toc,
map, zero*, cutout, info, extract,
inject_cd*, inject_dvd*, install*, cdvd_info, cdvd_info2,
poweroff, initialize*, backup_toc, restore_toc*, diag,
modify*, copy_hdd*, modify_header*

Use: hdl_dump.exe command
to show "command" help.

Warning: Commands, marked with * (asterisk) does write on the HDD
         and could cause corruption. Use with care.

License: You are only allowed to use this program with a software
         you legally own. Use at your own risk.

**Comment by bignaux** [comment]
Yes you're using old version, please try on https://github.com/ps2homebrew/hdl-dump/releases/tag/latest .

**Comment by rs1n** [comment]
Just tested with the latest binary from your link, and got the same result for SCUS97558 (Jak and Daxter - Lost Frontier):

hdl_dump cdvd_info "e:\PS2ISO\Jak and Daxter - TLF.iso"
e:\PS2ISO\Jak and Daxter - TLF.iso: not a Playstation CD-ROM/DVD-ROM.

MD5 using Microsoft's fciv.exe:

fciv.exe "e:\PS2ISO\Jak and Daxter - TLF.iso"
//
// File Checksum Integrity Verifier version 2.05.
//
5b946ee63898b1a78279ff4e64819d22 e:\ps2iso\jak and daxter - tlf.iso

EDIT: The same error also shows up for cdvd_info2

**Comment by bignaux** [comment]
Just type the game ID in manually and it should work, example
hdl_dump inject_dvd hdd1: "Gran Turismo 3" c:\gt3.iso SCES_xxx.xx

**Comment by rs1n** [comment]
There is no issue installing the affected titles manually. I was mainly reporting on the behavior in the cdvd_info2 command.

**Comment by bignaux** [comment]
Oki , thanks for your bug report.

**Comment by AKuHAK** [comment]
@rs1n fixed in #41 for Jak and Daxter - Lost Frontier
Regarding Dynasty Warriors 2, it seems that HDL-dump cannot correctly pass multitrack PS2 CD titles. I added in #41 also notification that game should be converted into a single cue/bin (with CDMage for example). hdl-dump can install only track1, so you probably missed some audio in-game. Actually, I don't even know if multitrack titles are correctly parsed by OPL. Anyway, now cdvd_info will not fail without reason, but I don't add a check for multitrack at the installation stage.
List of multitrack games:
```
Dance Factory (Europe) (Beta)
Dance Factory (Europe) (En,Fr,De,Es,It)
Dance Factory (USA)
Dance Factory (USA) (Demo)
Dynasty Warriors 2 (Europe)
Dynasty Warriors 2 (France)
Dynasty Warriors 2 (Germany)
Dynasty Warriors 2 (USA)
Frequency (USA) (Demo)
Operation WinBack (Europe) (En,Fr,De)
Oretachi Geesen Zoku - Karate-dou (Japan)
Oretachi Geesen Zoku - Time Pilot (Japan)
PrePre Plus 004 - 2000 June (Japan) (Disc 2)
Shin Sangoku Musou (Japan)
Shin Sangoku Musou (Japan) (Taikenban)
Simple 2000 Series Vol. 12 - The Quiz 20,000-mon (Japan)
```
